src/test/regress/spec/isolation_add_remove_node.spec

```diff
@@ -107,43 +107,43 @@ step "s2-commit"
 	COMMIT;
 }
 
-# session 1 adds a node, session 2 removes it, should be ok
+// session 1 adds a node, session 2 removes it, should be ok
```

Comment:
@JelteF and @SaitTalhaNisanci, I get the following error when I execute isolation tests locally:

```
syntax error at line 114: unexpected character "/"
```

Do you have any idea why? All the isolation tests with `//` comment fails. What was the reason to change this?

Reply:
@onderkalaci `make check-isolation` works fine in my local. 

The reason for this change was that we have many common code in isolation tests and we thought of using c preprocessor to handle this. We use CPP to generate actual isolation files before testing. And `//` are ignored since it is a comment in CPP.

What do you run locally?

Reply:
```
gcc --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 11.0.0 (clang-1100.0.33.17)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

And, MacOS Majove 10.14.6 (18G2022)

Reply:
I think he meant what command to actually run the tests? Also `make check-isolation`?

Reply:
> I think he meant what command to actually run the tests? Also make check-isolation?

Yes, make check-isolation

Reply:
A quick workaround would probably be installing the gcc version of cpp and using that one in the Makefile here: https://github.com/citusdata/citus/pull/3062/files#diff-f89c636652a95930b80fdfc4290d15c1R84. I guess it's something weird about the `clang` version of `cpp` that it doesn't always strip comments away. The reason it was changed from `#` comments to `//` is that `#` is interpreted as a preprocessor command, like `#include`.

Reply:
Ran into this today as well.

Pretty frustrating we break dev infra on a regular basis for OSX.

I think when we are changing build infra someone with a Mac should run it at least locally before we merge these things.

Would be great if we can treat this as an outage and fix it ASAP as it is non-trivial to change test output without being able to run these tests in your dev env. Especially with a code freeze upcoming.

Asking an annoying question, but what would it take to revert this PR?

Reply:
This is fixed now in #3706. 
> I think when we are changing build infra someone with a Mac should run it at least locally before we merge these things.

Yes I agree

> Would be great if we can treat this as an outage and fix it ASAP

Yes I agree. I think this was not done because it was weekend when this was first reported. Then during the week it was not picked up. I guess because it wasn't a problem for @onderkalaci at that point anymore.

Reply:
Thanks @JelteF @SaitTalhaNisanci and @onderkalaci for chipping in.